packages/build_packages.sh

```diff
@@ -10,14 +10,17 @@ cd "$(dirname "$0")"
 
 # ensure ordered by dependency dag
 while read -r package_dir; do
-    printf "\n========= RELEASING PACKAGE ${package_dir} =========\n\n"
-
-    cd "$package_dir"
-
-    make clean
-    make commit-version
-    make dist
-    make new-version
-
-    cd ..
+	if [ -n "$package_dir" ]
+	then
```

Comment:
It's the same thing, but maybe https://github.com/galaxyproject/galaxy/blob/dev/packages/test.sh#L36 saves you from having to indent the whole block of code.

Reply:
`if [ ! -z...` was my first attempt - shellcheck suggested this instead. But you're right - it's better to get rid of the indentation.